src/Module.zig

```diff
@@ -1104,6 +1109,10 @@ fn astGenAndAnalyzeDecl(self: *Module, decl: *Decl) !bool {
             var inst_table = Scope.Block.InstTable.init(self.gpa);
             defer inst_table.deinit();
 
+            var shared = try self.gpa.create(Scope.Block.Shared);
+            defer self.gpa.destroy(shared);
```

Comment:
if this were heap allocated it should be `const` instead of `var` but this should be stack allocated instead. See directly above we stack allocate inst_table and then pass a pointer to it into Scope.Block

Reply:
Ah gotcha, It looked to me that the above one was heap allocated because it was using gpa.